test/e2e/capi_test.go

```diff
@@ -91,16 +91,15 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 		})
 	})
 
-	// TODO: fix and enable
-	// Context("Running the self-hosted spec", func() {
-	// 	capi_e2e.SelfHostedSpec(context.TODO(), func() capi_e2e.SelfHostedSpecInput {
-	// 		return capi_e2e.SelfHostedSpecInput{
-	// 			E2EConfig:             e2eConfig,
-	// 			ClusterctlConfigPath:  clusterctlConfigPath,
-	// 			BootstrapClusterProxy: bootstrapClusterProxy,
-	// 			ArtifactFolder:        artifactFolder,
-	// 			SkipCleanup:           skipCleanup,
-	// 		}
-	// 	})
-	// })
+	Context("Running the self-hosted spec", func() {
```

Comment:
/hold

Let's make sure we don't merge this PR until we have a passing signal on the capi e2e test or undo this change otherwise it will break the periodic job

Reply:
For sure! Hopefully, it passes and we have one more test enabled.